RELEASE/scripts/autoscend.ash

```diff
@@ -250,7 +250,8 @@ boolean auto_unreservedAdvRemaining()
 	}
 
 	// automatically calculate adv to reserve at end of day
-	// free fights & free crafting require at least 1 adventure.
+	// free crafting require at least 1 adventure to do.
```

Comment:
Does free crafting require an adventure? It's free. If we do non-free crafting then this makes more sense, but perhaps note what sorts of things we craft. Also recommend rewrite to

```
To enter free fights, we need at least 1 adventure remaining. Dying costs an adventure, so we reserve 2 adventures so the user can manually complete the remaining fights, even if we lose.
```

Reply:
Yes, it require an adventure.
Kol refuses to allow you to perform free crafting if you have 0 adventures left.
You must have at least 1 adv left. after which 0 of them will be consumed by the free crafting.

This only applies to items that originally require adv to craft in the first place. Items that required 0 adv to craft can be crafted at 0 adv. But not items that originally cost 1 adv to craft before the cost is reduced to free.

I had long ago submitted a bug report and was told they will not fix it.

>Also recommend rewrite to
What is wrong with it? what is your suggested fix for that sentence?

Reply:
For some reason I looked at that comment you quoted and thought "isn't this what I wrote?". but no, it is not, it is the suggested alternative to what I wrote.

change made f917540